Jenkinsfile

```diff
@@ -132,7 +170,7 @@ pipeline {
                             name: STAGE_NAME,
                             title: 'Downloading',
                             summary: ':arrow_down: Downloading RTI Connext DDS libraries...',
```

Comment:
Is this text seen by customers? We are slowly removing "DDS" from "RTI Connext DDS," but not necessarily in code/directory names, etc. If these are the only instances, or if you want to, remove "DDS" from these. But if that makes things inconsistent, and searching/replacing all is too big a project, then never mind.

Reply:
Alright! I'll remove them! Yes, these messages are seen by the costumers.